grafana/mattermost-web-app-metrics.json

```diff
@@ -262,12 +261,12 @@
           "disableTextWrap": false,
           "editorMode": "code",
           "exemplar": false,
-          "expr": "histogram_quantile(0.75, sum by(le) (increase(mattermost_webapp_channel_switch_bucket[24h])))",
+          "expr": "histogram_quantile($percentile / 100, sum by(le) (increase(mattermost_webapp_channel_switch_bucket{agent=~\"$agent\",platform=~\"$platform\"}[$__range])))",
           "format": "time_series",
           "fullMetaSearch": false,
           "includeNullMetadata": true,
           "instant": false,
-          "legendFormat": "75th Percentile",
+          "legendFormat": "[[percentile]]th Percentile",
```

Comment:
Isn't the legendFormat template as `{{}}` rather than `[[]]`?

Reply:
It might be that if the identifier in them comes from a label on the series, but the square brackets work here when the value comes from a variable. Maybe there's some other detail I'm missing here, but this seems to work since this is directly taken from the Web App Metrics dashboard